yb-voyager/cmd/importSchemaYugabyteDB.go

```diff
@@ -37,6 +41,46 @@ func importSchemaInternal(exportDir string, importObjectList []string,
 		fmt.Printf("\nImporting %q\n\n", importObjectFilePath)
 		executeSqlFile(importObjectFilePath, importObjectType, skipFn)
 	}
+
+}
+
+/*
+Try re-executing each DDL from deffered list.
+If fails, silently avoid the error.
+Else remove from defferedSQLStmts list
+At the end, add the unsuccessful ones to a failedSqlStmts list and report to the user
+*/
+func importDefferedStatements() {
+	if len(defferedSqlStmts) == 0 {
+		return
+	}
+	log.Infof("Number of statements in defferedSQLStmts list: %d\n", len(defferedSqlStmts))
+
+	utils.PrintAndLog("\nExecuting the remaining SQL statements...\n\n")
+	maxIterations := len(defferedSqlStmts)
+	conn := newTargetConn()
+	defer conn.Close(context.Background())
```

Comment:
The arguments to the deferred function are evaluated/determined at the time the `defer` statement is processed. That means, the reference to the `conn` and `context.Background()` object at line 62 is saved and used later when the `Close()` function is invoked. For more info, visit https://go.dev/blog/defer-panic-and-recover and search for `arguments are evaluated when the defer statement is evaluated`

That means, if you create a new connection and assign it to `conn`, it will not be closed by the defer call. See https://go.dev/play/p/e0MisbNL0Ue

The fix will be to use a closure:
```
defer func() { conn.Close(context.Background() }()
```

See if there are similar issues elsewhere and fix them.

Reply:
Updated this line of code as per your comment. There are no more occurrences of `conn` i found where new connections are made on the same variable.